File: back-end/src/main/java/com/spring/restaurant/model/Order.java
package com.spring.restaurant.model;

import com.fasterxml.jackson.annotation.JsonIgnore;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import javax.persistence.*;

@Data
@NoArgsConstructor
@AllArgsConstructor
@Entity
@Table(name = "orders")
public class Order extends PublicData{

    @Column(name = "price")
    private int price;
    @Column(name = "image")
    private String img;

    @Column(name = "description")
    @Lob
    private String description;

    @JsonIgnore
    @ManyToOne
    @JoinColumn(name = "id_Category")
    private Category category;

}

File: front-end/src/app/model/order.ts
export class Order {

  id: number;
  name: string;
  data_create: Date;
  data_update: Date;
  description: string;
  img: string;
  price: number;


  constructor(id: number, name: string, data_create: Date, data_update: Date, description: string, img: string, price: number) {
    this.id = id;
    this.name = name;
    this.data_create = data_create;
    this.data_update = data_update;
    this.description = description;
    this.img = img;
    this.price = price;
  }
}
